umm/file/__init__.py

```diff
@@ -123,18 +105,19 @@ def convert(
         case _:
             template_version = _version_map(file_type, umm_version)
             template_file = (
-                importlib.resources.files("umm") /
-                "assets" /
-                "templates" /
-                file_type /
-                f"{template_version}.template"
+                importlib.resources.files("umm")
+                / "assets"
+                / "templates"
+                / file_type
+                / f"{template_version}.template"
             )
-            print(_version_map(file_type, umm_version))
             with open(template_file, mode="r", encoding="utf-8") as file:
                 template = Template(file.read())
             with open(output_file, "w", encoding="utf-8") as file:
                 file.write(template.render(umm=umm))
 
+            print(f"Successfully converted {input_file} to {output_file}.")
```

Comment:
We should probably think about things like this for a production release - you may not want the application to output text by default for the purposes of automation 🤔